Makefile

```diff
@@ -3,7 +3,7 @@ PORT ?= 3128
 
 SHELL = /bin/bash
 
-DESTDIR ?= /
+PREFIX ?= /usr
```

Comment:
The default should really be `usr/local`, don't you think? Under that assumption, Debian packaging scripts will automatically set `PREFIX=/usr` . That does mean we'll need to template the `.service` file, and/or just assume that it is in `$PATH` and not give an absolute path.